src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -10,4 +10,8 @@
     Optional<CinemaHall> get(Long id);
 
     List<CinemaHall> getAll();
+
+    boolean update(CinemaHall cinemaHall);
+
+    boolean delete(Long id);
```

Comment:
The return type of the `delete` method should be `boolean` instead of `Boolean` to avoid unnecessary boxing.